### ISSUE TITLE
The specified location for moving files to the desktop is wrong

### ISSUE THREAD

**Issue by louies0623** [helloSystem/Filer#176]
[PC-1-screen0.webm](https://github.com/helloSystem/ISO/assets/44593430/95fc152e-c741-4e01-9e0e-4c9614c4d7d5)

It should be placed where the mouse.

**Comment by probonopd** [comment]
Yes. Also, the icon position in the window should persist closing and opening the window.
